test/otel/testdata/ocb_build_script.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Should we `set -e` at the top of this script so that if any command in the script fails we exit out early and not continue?

Reply:
sure, I don't see why not